nuxeo-core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/jdbc/SQLInfo.java

```diff
@@ -97,7 +99,11 @@
 
     private String selectChildrenIdsAndTypesSql;
 
+    private String selectChildrenIdsAndTypesSqlExcludeSpecialChildren;
+    
     private String selectComplexChildrenIdsAndTypesSql;
+    
+    private String selectComplexChildrenIdsAndTypesSqlIncludeSpecialChildren;
```

Comment:
The two variables have inconsistent naming, one is "exclude" and the other "include"... Please make them consistent, both with "exclude"

Reply:
I'll make both exclude then.
This was made to reflect the query passed.
The exclude has a NOT IN clause
While the include has just an IN